src/test/scala/de/zalando/beard/parser/BeardTemplateParserSpec.scala

```diff
@@ -167,6 +167,16 @@ class BeardTemplateParserSpec extends FunSpec with Matchers {
     }
   }
 
+  describe("when parsing a string that contains a comment") {
```

Comment:
In addition to that, I have added another test that verifies if the Beard comment is properly skipped:
```beard
Hello {{# text {{ interpolation }} #}} world
```